src/main/java/org/opentripplanner/updater/alerts/GtfsEnhancedRealtimeAlertsUpdater.java

```diff
@@ -77,6 +77,10 @@ public String toString() {
         JsonNode json = mapper.readTree(jsonString);
 
         for (JsonNode node: json.get("entity")) {
+            if (!node.hasNonNull("alert") || !node.get("alert").hasNonNull("informed_entity")) {
```

Comment:
If you split this up, you could avoid fetching the `alert` node twice:

```java
if (!node.hasNotNull("alert")) { continue; }
JsonNode alert = node.get("alert");
if (!alert.hasNonNull("informed_entity")) { continue; }
...
for (JsonNone ie: alert.get("informed_entity")) {
    ...
}
```

Reply:
Refactored